lesson15/index.html

```diff
@@ -17,11 +17,18 @@
       <ul id="js-lists"></ul>
     </div>
     <div id="js-modal" class="modal center mt-20 p-20 hidden">
-      <div class="modal__form--number txt-center">
-        <label for="number">Number:</label>
-        <input type="number" id="js-number" class="input">
-      </div>
-      <button id="js-request" class="modal__btn center button mt-20">Request the data</button>
+      <form id="js-form" class="modal__form txt-center">
+        <p class="txt-center mt-20">数字と名前を入れてください</p>
+        <div class="form-item">
+          <label for="js-number">数字:</label>
+          <input type="number" id="js-number" class="p-5">
+        </div>
+        <div class="form-item">
+          <label for="js-name">名前:</label>
+          <input type="text" id="js-name" class="p-5">
+        </div>
```

Comment:
It is a problem if the form is sent without any input, so make it a required field.😄

It is possible to implement this on the JS side as well, such as disabling the button until the input is done, or sending a message if the input is not done, but this is not written as a specification for this problem, so I thought it would be easiest to implement it on the HTML side IMO.
```suggestion
        <div class="form-item">
          <label for="js-number">数字:</label>
          <input type="number" id="js-number" class="p-5" required>
        </div>
        <div class="form-item">
          <label for="js-name">名前:</label>
          <input type="text" id="js-name" class="p-5" required>
        </div>
```

Reply:
Thanks! It's important things but I forgot it...
I added required attribute on inputs as you taught me! And also I added asterisk on the label to let users know that the field is required.
[d43e85e](https://github.com/yuka830/js-lessons/pull/17/commits/d43e85e912bae572a6db2949654423268720dbd2)